index.html

```diff
@@ -12,11 +12,11 @@
   <div class="content">
     <div class="reasons">
       <div class="section">
-        <h1 style="font-weight: normal;">Passerine <a href="https://github.com/vrtbl/passerine"><img src="https://raw.githubusercontent.com/vrtbl/passerine/master/Logo.png" class="inline"></img></a></h1>
-        <h3>A small extensible programming language designed for concise expression with little code.</h3>
+        <h1 style="font-weight: bold;">Passerine <a href="https://github.com/vrtbl/passerine"><img src="https://raw.githubusercontent.com/vrtbl/passerine/master/Logo.png" class="inline"></img></a></h1>
```

Comment:
A bit on the fence about this 🤔 (changing font weight to bold) - but that's something for another time.